.github/workflows/test.yml

```diff
@@ -18,15 +18,14 @@ jobs:
       - uses: actions/setup-python@v2
         with:
           python-version: "3.9"
+      - uses: Gr1N/setup-poetry@v8
 
-      - name: Install dependencies
+      - name: Install dependencies and app
         run: |
-          python -m pip install .[dev]
-      - name: Install the app
-        run: |
-          python -m pip install --no-deps -e .
+          poetry install --with cdk
```

Comment:
when installing with poetry, do you need to`poetry install .[tests]` or anything similar to signify the optional dependencies?

Reply:
So, the tests dependency is automatically installed with `poetry install` (this can be changed if you don't like that default.) The cdk section is not automatically installed, which is why I needed to include it here with `--with`. 

The test dependencies can be excluded with `poetry install --without tests`.

Reply:
Nice, thanks for clarifying, I didn't know that!